source/OptimaJet.DWKit.StarterApplication/Services/OrganizationInviteRequestService.cs

```diff
@@ -10,42 +13,55 @@ namespace OptimaJet.DWKit.StarterApplication.Services
 {
     public class OrganizationInviteRequestService : IOrganizationInviteRequestService
     {
+        private readonly ITranslator translator;
         protected readonly IJobRepository<OrganizationInviteRequest> requestRepository;
         protected readonly IJobRepository<Email> emailRepository;
+        private readonly IJobRepository<UserRole> userRolesRepository;
         protected readonly OrganizationInviteRequestSettings settings;
-        protected readonly IClient bugsnagClient;
         public OrganizationInviteRequestService(
+            ITranslator translator,
             IOptions<OrganizationInviteRequestSettings> options,
             IJobRepository<OrganizationInviteRequest> requestRepository,
             IJobRepository<Email> emailRepository,
-            IClient bugsnagClient)
+            IJobRepository<UserRole> userRolesRepository)
         {
+            this.translator = translator;
             this.requestRepository = requestRepository;
             this.emailRepository = emailRepository;
-            this.bugsnagClient = bugsnagClient;
+            this.userRolesRepository = userRolesRepository;
             this.settings = options.Value;
         }
 
         public void Process(OrganizationInviteRequestServiceData data)
         {
             var request = requestRepository.GetAsync(data.Id).Result;
-            var email = new Email
+            var superAdmins = userRolesRepository.Get()
+                .Include(ur => ur.User)
+                .Include(ur => ur.Role)
+                .Where(ur =>  ur.Role.RoleName == RoleName.SuperAdmin
+                       && ur.User.EmailNotification == true
+                )
```

Comment:
I am wondering if the `EmailNotification` should only apply to AppBuilder role and not SuperAdmin (and OrgAdmin?) role. To me, it seems like we don't want SuperAdmin to opt out of email since these are things they need to deal with.

Reply:
After we talked about it, we shouldn't have SuperAdmin or OrgAdmin opt out of email for now.